lib/Field/styles.scss

```diff
@@ -314,7 +313,7 @@ $field-data-p-line-height: 1.688rem !default;
   }
 }
 
-.field.has-formatting {
+.has-formatting .field__content {
```

Comment:
This still looks right to me but I'd like a second opinion here, as this could be a potentially big change if I got it wrong!